01-getting-started/quick-start.ipynb

```diff
@@ -17,8 +17,10 @@
     "This notebook covers the following steps:\n",
```

Comment:
<p>Include an image of the created chart</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>